src/electron/main.js

```diff
@@ -77,16 +77,13 @@ app.whenReady().then(async () => {
 
     // DevTools configuration
     mb.window.webContents.on('devtools-opened', () => {
```

Comment:
I'm not too sure what type of experience you're looking for with these changes... 

To me, this is less useful DX...  My local dev/debug flow uses Cmd+Opt+I a lot, along with resizing the window so that I can see the Chrome console, network tab or local storage in full.  These changes for me would make it much harder to locally dev/debug

Reply:
Ah, I see! I use the devtools detached on a new window, but what you are saying makes sense for you usage style.

Reply:
Maybe a solution with both might be useful to everyone then, aka, just adding the setAlwaysOnTop?